compose/wordpress.md

```diff
@@ -0,0 +1,145 @@
+---
+description: Getting started with Compose and WordPress
+keywords: documentation, docs,  docker, compose, orchestration, containers
+title: "Quickstart: Compose and WordPress"
+---
+
+You can use Docker Compose to easily run WordPress in an isolated environment
+built with Docker containers. This quick-start guide demonstrates how to use
+Compose to set up and run WordPress. Before starting, you'll need to have
+[Compose installed](install.md).
+
+### Define the project
+
+1. Create an empty project directory.
```

Comment:
These numbered lists need an extra space. Two spaces after the period, to line up with the lines underneath.